hagrid/reservations/export_pdf.py

```diff
@@ -125,18 +125,33 @@ def generate_qr_code(link: str):
     return qr.make_image()
 
 
-def generate_collection_list(reservation: Reservation):
-    variation_dict = {}
-    for res_part in ReservationPart.objects.all().filter(reservation=reservation):
-        for p in ReservationPosition.objects.all().filter(part=res_part):
-            if p.variation in variation_dict:
-                variation_dict[p.variation] += p.amount
-            else:
-                variation_dict[p.variation] = p.amount
-    article_list = []
-    for entry in variation_dict:
-        article_list.append((entry, variation_dict[entry]))
-    return sorted(article_list, key=lambda t: str(t[0]))
+def generate_collection_list(reservation: Reservation, distinct_required = False):
+    groups = []
+    titles = []
+
+    part_set = []
```

Comment:
I'd like a better name to make clear that this is a list of sections each being a list of reservation parts
```suggestion
    sections = []
```